db/migrate/20230926102535_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :bio
+      t.integer :posts_counter
+
+      t.timestamps
```

Comment:
- [ ] I'm very sorry to inform you that your app isn't complete yet, and still has a bit to go through. Please start by removing the duplicated migration files as requested by the previous reviewer. Follow the steps down below:
1. run `rails db:migrate VERSION=0` This will rollback all the migrations and clear the schema, to ensure a clean database when migrating again
2. Revise your migration files' order and columns for each table. 
3. Delete all of the unused and non-running migration files
4. Run `rails db:migrate` to rebuild the database schema
This will help you solve all the conflict errors in your app and run correctly. 🙏